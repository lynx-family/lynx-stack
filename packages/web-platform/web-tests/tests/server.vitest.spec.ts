import { test, expect, describe } from 'vitest';
import { createLynxView } from '@lynx-js/web-core-server';
import path from 'node:path';
import { readFile } from 'fs/promises';
import { fileURLToPath } from 'node:url';
const __filename = fileURLToPath(import.meta.url);
const __dirname = path.dirname(__filename);

async function readTemplate(name: string) {
  const file = await readFile(
    path.join(__dirname, `../dist/${name}/index.web.json`),
    'utf-8',
  );
  const rawTemplate = JSON.parse(file);
  return rawTemplate as any;
}
describe('server-tests', () => {
  test('basic-performance-div-10000', async ({ task }) => {
    const testName = task.name;
    const rawTemplate = await readTemplate(testName);
    const lynxView = createLynxView({
      browserConfig: {
        pixelRatio: 1,
        pixelHeight: 800,
        pixelWidth: 375,
      },
      tagMap: {},
      initData: {},
      globalProps: {},
      template: rawTemplate,
    });

    const html = await lynxView.renderToString();
    expect(html).toMatchSnapshot();
  });
});
