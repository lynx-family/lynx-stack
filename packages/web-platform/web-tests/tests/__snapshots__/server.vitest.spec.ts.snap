// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`server-tests > basic-performance-div-10 1`] = `
"<lynx-view url="/dist/basic-performance-div-10/index.web.json" ssr height="auto" width="auto" style="width:100vw; max-width: 500px;" ssr-encode-data="%7B%22__opcodes%22:%5B0,%5B%22__Card__:__snapshot_67658_d2b44_1%22,-2,%5B%7B%22ssrID%22:%222%22%7D%5D%5D,2,%22values%22,%5B%22target-0%22%5D,1,0,%5B%22__Card__:__snapshot_67658_d2b44_1%22,-3,%5B%7B%22ssrID%22:%223%22%7D%5D%5D,2,%22values%22,%5B%22target-1%22%5D,1,0,%5B%22__Card__:__snapshot_67658_d2b44_1%22,-4,%5B%7B%22ssrID%22:%224%22%7D%5D%5D,2,%22values%22,%5B%22target-2%22%5D,1,0,%5B%22__Card__:__snapshot_67658_d2b44_1%22,-5,%5B%7B%22ssrID%22:%225%22%7D%5D%5D,2,%22values%22,%5B%22target-3%22%5D,1,0,%5B%22__Card__:__snapshot_67658_d2b44_1%22,-6,%5B%7B%22ssrID%22:%226%22%7D%5D%5D,2,%22values%22,%5B%22target-4%22%5D,1,0,%5B%22__Card__:__snapshot_67658_d2b44_1%22,-7,%5B%7B%22ssrID%22:%227%22%7D%5D%5D,2,%22values%22,%5B%22target-5%22%5D,1,0,%5B%22__Card__:__snapshot_67658_d2b44_1%22,-8,%5B%7B%22ssrID%22:%228%22%7D%5D%5D,2,%22values%22,%5B%22target-6%22%5D,1,0,%5B%22__Card__:__snapshot_67658_d2b44_1%22,-9,%5B%7B%22ssrID%22:%229%22%7D%5D%5D,2,%22values%22,%5B%22target-7%22%5D,1,0,%5B%22__Card__:__snapshot_67658_d2b44_1%22,-10,%5B%7B%22ssrID%22:%2210%22%7D%5D%5D,2,%22values%22,%5B%22target-8%22%5D,1,0,%5B%22__Card__:__snapshot_67658_d2b44_1%22,-11,%5B%7B%22ssrID%22:%2211%22%7D%5D%5D,2,%22values%22,%5B%22target-9%22%5D,1%5D%7D"><template shadowrootmode="open"><style>@import url("/fp-only.css");
 [lynx-default-display-linear="false"] * {
    --lynx-display: flex;
    --lynx-display-toggle: var(--lynx-display-flex);
  }
[lynx-default-overflow-visible="true"] x-view{
    overflow: visible;
  }</style><style></style><div lynx-tag="page" lynx-unique-id="1" lynx-css-id="0" lynx-component-id="0" part="page" lynx-default-overflow-visible="true"><div lynx-tag="div" lynx-unique-id="2" style="height:100px;width:100px;background:pink;" id="target-0"></div><div lynx-tag="div" lynx-unique-id="3" style="height:100px;width:100px;background:pink;" id="target-1"></div><div lynx-tag="div" lynx-unique-id="4" style="height:100px;width:100px;background:pink;" id="target-2"></div><div lynx-tag="div" lynx-unique-id="5" style="height:100px;width:100px;background:pink;" id="target-3"></div><div lynx-tag="div" lynx-unique-id="6" style="height:100px;width:100px;background:pink;" id="target-4"></div><div lynx-tag="div" lynx-unique-id="7" style="height:100px;width:100px;background:pink;" id="target-5"></div><div lynx-tag="div" lynx-unique-id="8" style="height:100px;width:100px;background:pink;" id="target-6"></div><div lynx-tag="div" lynx-unique-id="9" style="height:100px;width:100px;background:pink;" id="target-7"></div><div lynx-tag="div" lynx-unique-id="10" style="height:100px;width:100px;background:pink;" id="target-8"></div><div lynx-tag="div" lynx-unique-id="11" style="height:100px;width:100px;background:pink;" id="target-9"></div></div></template></lynx-view>"
`;

exports[`server-tests > basic-performance-nest-level-100 1`] = `
"<lynx-view url="/dist/basic-performance-nest-level-100/index.web.json" ssr height="auto" width="auto" style="width:100vw; max-width: 500px;" ssr-encode-data="%7B%22__opcodes%22:%5B0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-2,%5B%7B%22ssrID%22:%222%22%7D%5D%5D,2,%22values%22,%5B%22target-100%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-3,%5B%7B%22ssrID%22:%223%22%7D%5D%5D,2,%22values%22,%5B%22target-99%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-4,%5B%7B%22ssrID%22:%224%22%7D%5D%5D,2,%22values%22,%5B%22target-98%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-5,%5B%7B%22ssrID%22:%225%22%7D%5D%5D,2,%22values%22,%5B%22target-97%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-6,%5B%7B%22ssrID%22:%226%22%7D%5D%5D,2,%22values%22,%5B%22target-96%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-7,%5B%7B%22ssrID%22:%227%22%7D%5D%5D,2,%22values%22,%5B%22target-95%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-8,%5B%7B%22ssrID%22:%228%22%7D%5D%5D,2,%22values%22,%5B%22target-94%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-9,%5B%7B%22ssrID%22:%229%22%7D%5D%5D,2,%22values%22,%5B%22target-93%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-10,%5B%7B%22ssrID%22:%2210%22%7D%5D%5D,2,%22values%22,%5B%22target-92%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-11,%5B%7B%22ssrID%22:%2211%22%7D%5D%5D,2,%22values%22,%5B%22target-91%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-12,%5B%7B%22ssrID%22:%2212%22%7D%5D%5D,2,%22values%22,%5B%22target-90%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-13,%5B%7B%22ssrID%22:%2213%22%7D%5D%5D,2,%22values%22,%5B%22target-89%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-14,%5B%7B%22ssrID%22:%2214%22%7D%5D%5D,2,%22values%22,%5B%22target-88%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-15,%5B%7B%22ssrID%22:%2215%22%7D%5D%5D,2,%22values%22,%5B%22target-87%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-16,%5B%7B%22ssrID%22:%2216%22%7D%5D%5D,2,%22values%22,%5B%22target-86%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-17,%5B%7B%22ssrID%22:%2217%22%7D%5D%5D,2,%22values%22,%5B%22target-85%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-18,%5B%7B%22ssrID%22:%2218%22%7D%5D%5D,2,%22values%22,%5B%22target-84%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-19,%5B%7B%22ssrID%22:%2219%22%7D%5D%5D,2,%22values%22,%5B%22target-83%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-20,%5B%7B%22ssrID%22:%2220%22%7D%5D%5D,2,%22values%22,%5B%22target-82%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-21,%5B%7B%22ssrID%22:%2221%22%7D%5D%5D,2,%22values%22,%5B%22target-81%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-22,%5B%7B%22ssrID%22:%2222%22%7D%5D%5D,2,%22values%22,%5B%22target-80%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-23,%5B%7B%22ssrID%22:%2223%22%7D%5D%5D,2,%22values%22,%5B%22target-79%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-24,%5B%7B%22ssrID%22:%2224%22%7D%5D%5D,2,%22values%22,%5B%22target-78%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-25,%5B%7B%22ssrID%22:%2225%22%7D%5D%5D,2,%22values%22,%5B%22target-77%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-26,%5B%7B%22ssrID%22:%2226%22%7D%5D%5D,2,%22values%22,%5B%22target-76%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-27,%5B%7B%22ssrID%22:%2227%22%7D%5D%5D,2,%22values%22,%5B%22target-75%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-28,%5B%7B%22ssrID%22:%2228%22%7D%5D%5D,2,%22values%22,%5B%22target-74%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-29,%5B%7B%22ssrID%22:%2229%22%7D%5D%5D,2,%22values%22,%5B%22target-73%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-30,%5B%7B%22ssrID%22:%2230%22%7D%5D%5D,2,%22values%22,%5B%22target-72%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-31,%5B%7B%22ssrID%22:%2231%22%7D%5D%5D,2,%22values%22,%5B%22target-71%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-32,%5B%7B%22ssrID%22:%2232%22%7D%5D%5D,2,%22values%22,%5B%22target-70%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-33,%5B%7B%22ssrID%22:%2233%22%7D%5D%5D,2,%22values%22,%5B%22target-69%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-34,%5B%7B%22ssrID%22:%2234%22%7D%5D%5D,2,%22values%22,%5B%22target-68%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-35,%5B%7B%22ssrID%22:%2235%22%7D%5D%5D,2,%22values%22,%5B%22target-67%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-36,%5B%7B%22ssrID%22:%2236%22%7D%5D%5D,2,%22values%22,%5B%22target-66%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-37,%5B%7B%22ssrID%22:%2237%22%7D%5D%5D,2,%22values%22,%5B%22target-65%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-38,%5B%7B%22ssrID%22:%2238%22%7D%5D%5D,2,%22values%22,%5B%22target-64%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-39,%5B%7B%22ssrID%22:%2239%22%7D%5D%5D,2,%22values%22,%5B%22target-63%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-40,%5B%7B%22ssrID%22:%2240%22%7D%5D%5D,2,%22values%22,%5B%22target-62%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-41,%5B%7B%22ssrID%22:%2241%22%7D%5D%5D,2,%22values%22,%5B%22target-61%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-42,%5B%7B%22ssrID%22:%2242%22%7D%5D%5D,2,%22values%22,%5B%22target-60%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-43,%5B%7B%22ssrID%22:%2243%22%7D%5D%5D,2,%22values%22,%5B%22target-59%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-44,%5B%7B%22ssrID%22:%2244%22%7D%5D%5D,2,%22values%22,%5B%22target-58%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-45,%5B%7B%22ssrID%22:%2245%22%7D%5D%5D,2,%22values%22,%5B%22target-57%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-46,%5B%7B%22ssrID%22:%2246%22%7D%5D%5D,2,%22values%22,%5B%22target-56%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-47,%5B%7B%22ssrID%22:%2247%22%7D%5D%5D,2,%22values%22,%5B%22target-55%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-48,%5B%7B%22ssrID%22:%2248%22%7D%5D%5D,2,%22values%22,%5B%22target-54%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-49,%5B%7B%22ssrID%22:%2249%22%7D%5D%5D,2,%22values%22,%5B%22target-53%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-50,%5B%7B%22ssrID%22:%2250%22%7D%5D%5D,2,%22values%22,%5B%22target-52%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-51,%5B%7B%22ssrID%22:%2251%22%7D%5D%5D,2,%22values%22,%5B%22target-51%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-52,%5B%7B%22ssrID%22:%2252%22%7D%5D%5D,2,%22values%22,%5B%22target-50%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-53,%5B%7B%22ssrID%22:%2253%22%7D%5D%5D,2,%22values%22,%5B%22target-49%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-54,%5B%7B%22ssrID%22:%2254%22%7D%5D%5D,2,%22values%22,%5B%22target-48%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-55,%5B%7B%22ssrID%22:%2255%22%7D%5D%5D,2,%22values%22,%5B%22target-47%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-56,%5B%7B%22ssrID%22:%2256%22%7D%5D%5D,2,%22values%22,%5B%22target-46%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-57,%5B%7B%22ssrID%22:%2257%22%7D%5D%5D,2,%22values%22,%5B%22target-45%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-58,%5B%7B%22ssrID%22:%2258%22%7D%5D%5D,2,%22values%22,%5B%22target-44%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-59,%5B%7B%22ssrID%22:%2259%22%7D%5D%5D,2,%22values%22,%5B%22target-43%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-60,%5B%7B%22ssrID%22:%2260%22%7D%5D%5D,2,%22values%22,%5B%22target-42%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-61,%5B%7B%22ssrID%22:%2261%22%7D%5D%5D,2,%22values%22,%5B%22target-41%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-62,%5B%7B%22ssrID%22:%2262%22%7D%5D%5D,2,%22values%22,%5B%22target-40%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-63,%5B%7B%22ssrID%22:%2263%22%7D%5D%5D,2,%22values%22,%5B%22target-39%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-64,%5B%7B%22ssrID%22:%2264%22%7D%5D%5D,2,%22values%22,%5B%22target-38%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-65,%5B%7B%22ssrID%22:%2265%22%7D%5D%5D,2,%22values%22,%5B%22target-37%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-66,%5B%7B%22ssrID%22:%2266%22%7D%5D%5D,2,%22values%22,%5B%22target-36%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-67,%5B%7B%22ssrID%22:%2267%22%7D%5D%5D,2,%22values%22,%5B%22target-35%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-68,%5B%7B%22ssrID%22:%2268%22%7D%5D%5D,2,%22values%22,%5B%22target-34%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-69,%5B%7B%22ssrID%22:%2269%22%7D%5D%5D,2,%22values%22,%5B%22target-33%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-70,%5B%7B%22ssrID%22:%2270%22%7D%5D%5D,2,%22values%22,%5B%22target-32%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-71,%5B%7B%22ssrID%22:%2271%22%7D%5D%5D,2,%22values%22,%5B%22target-31%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-72,%5B%7B%22ssrID%22:%2272%22%7D%5D%5D,2,%22values%22,%5B%22target-30%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-73,%5B%7B%22ssrID%22:%2273%22%7D%5D%5D,2,%22values%22,%5B%22target-29%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-74,%5B%7B%22ssrID%22:%2274%22%7D%5D%5D,2,%22values%22,%5B%22target-28%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-75,%5B%7B%22ssrID%22:%2275%22%7D%5D%5D,2,%22values%22,%5B%22target-27%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-76,%5B%7B%22ssrID%22:%2276%22%7D%5D%5D,2,%22values%22,%5B%22target-26%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-77,%5B%7B%22ssrID%22:%2277%22%7D%5D%5D,2,%22values%22,%5B%22target-25%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-78,%5B%7B%22ssrID%22:%2278%22%7D%5D%5D,2,%22values%22,%5B%22target-24%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-79,%5B%7B%22ssrID%22:%2279%22%7D%5D%5D,2,%22values%22,%5B%22target-23%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-80,%5B%7B%22ssrID%22:%2280%22%7D%5D%5D,2,%22values%22,%5B%22target-22%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-81,%5B%7B%22ssrID%22:%2281%22%7D%5D%5D,2,%22values%22,%5B%22target-21%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-82,%5B%7B%22ssrID%22:%2282%22%7D%5D%5D,2,%22values%22,%5B%22target-20%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-83,%5B%7B%22ssrID%22:%2283%22%7D%5D%5D,2,%22values%22,%5B%22target-19%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-84,%5B%7B%22ssrID%22:%2284%22%7D%5D%5D,2,%22values%22,%5B%22target-18%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-85,%5B%7B%22ssrID%22:%2285%22%7D%5D%5D,2,%22values%22,%5B%22target-17%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-86,%5B%7B%22ssrID%22:%2286%22%7D%5D%5D,2,%22values%22,%5B%22target-16%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-87,%5B%7B%22ssrID%22:%2287%22%7D%5D%5D,2,%22values%22,%5B%22target-15%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-88,%5B%7B%22ssrID%22:%2288%22%7D%5D%5D,2,%22values%22,%5B%22target-14%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-89,%5B%7B%22ssrID%22:%2289%22%7D%5D%5D,2,%22values%22,%5B%22target-13%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-90,%5B%7B%22ssrID%22:%2290%22%7D%5D%5D,2,%22values%22,%5B%22target-12%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-91,%5B%7B%22ssrID%22:%2291%22%7D%5D%5D,2,%22values%22,%5B%22target-11%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-92,%5B%7B%22ssrID%22:%2292%22%7D%5D%5D,2,%22values%22,%5B%22target-10%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-93,%5B%7B%22ssrID%22:%2293%22%7D%5D%5D,2,%22values%22,%5B%22target-9%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-94,%5B%7B%22ssrID%22:%2294%22%7D%5D%5D,2,%22values%22,%5B%22target-8%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-95,%5B%7B%22ssrID%22:%2295%22%7D%5D%5D,2,%22values%22,%5B%22target-7%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-96,%5B%7B%22ssrID%22:%2296%22%7D%5D%5D,2,%22values%22,%5B%22target-6%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-97,%5B%7B%22ssrID%22:%2297%22%7D%5D%5D,2,%22values%22,%5B%22target-5%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-98,%5B%7B%22ssrID%22:%2298%22%7D%5D%5D,2,%22values%22,%5B%22target-4%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-99,%5B%7B%22ssrID%22:%2299%22%7D%5D%5D,2,%22values%22,%5B%22target-3%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-100,%5B%7B%22ssrID%22:%22100%22%7D%5D%5D,2,%22values%22,%5B%22target-2%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-101,%5B%7B%22ssrID%22:%22101%22%7D%5D%5D,2,%22values%22,%5B%22target-1%22%5D,0,%5B%22__Card__:__snapshot_03f21_6db13_1%22,-102,%5B%7B%22ssrID%22:%22102%22%7D%5D%5D,2,%22values%22,%5B%22target-0%22%5D,3,%5B%5Bnull,-103,%5B%7B%22ssrID%22:%22103%22%7D%5D%5D,%220%22%5D,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1%5D%7D"><template shadowrootmode="open"><style>@import url("/fp-only.css");
 [lynx-default-display-linear="false"] * {
    --lynx-display: flex;
    --lynx-display-toggle: var(--lynx-display-flex);
  }
[lynx-default-overflow-visible="true"] x-view{
    overflow: visible;
  }</style><style></style><div lynx-tag="page" lynx-unique-id="1" lynx-css-id="0" lynx-component-id="0" part="page" lynx-default-overflow-visible="true"><div lynx-tag="div" lynx-unique-id="2" id="target-100"><div lynx-tag="div" lynx-unique-id="3" id="target-99"><div lynx-tag="div" lynx-unique-id="4" id="target-98"><div lynx-tag="div" lynx-unique-id="5" id="target-97"><div lynx-tag="div" lynx-unique-id="6" id="target-96"><div lynx-tag="div" lynx-unique-id="7" id="target-95"><div lynx-tag="div" lynx-unique-id="8" id="target-94"><div lynx-tag="div" lynx-unique-id="9" id="target-93"><div lynx-tag="div" lynx-unique-id="10" id="target-92"><div lynx-tag="div" lynx-unique-id="11" id="target-91"><div lynx-tag="div" lynx-unique-id="12" id="target-90"><div lynx-tag="div" lynx-unique-id="13" id="target-89"><div lynx-tag="div" lynx-unique-id="14" id="target-88"><div lynx-tag="div" lynx-unique-id="15" id="target-87"><div lynx-tag="div" lynx-unique-id="16" id="target-86"><div lynx-tag="div" lynx-unique-id="17" id="target-85"><div lynx-tag="div" lynx-unique-id="18" id="target-84"><div lynx-tag="div" lynx-unique-id="19" id="target-83"><div lynx-tag="div" lynx-unique-id="20" id="target-82"><div lynx-tag="div" lynx-unique-id="21" id="target-81"><div lynx-tag="div" lynx-unique-id="22" id="target-80"><div lynx-tag="div" lynx-unique-id="23" id="target-79"><div lynx-tag="div" lynx-unique-id="24" id="target-78"><div lynx-tag="div" lynx-unique-id="25" id="target-77"><div lynx-tag="div" lynx-unique-id="26" id="target-76"><div lynx-tag="div" lynx-unique-id="27" id="target-75"><div lynx-tag="div" lynx-unique-id="28" id="target-74"><div lynx-tag="div" lynx-unique-id="29" id="target-73"><div lynx-tag="div" lynx-unique-id="30" id="target-72"><div lynx-tag="div" lynx-unique-id="31" id="target-71"><div lynx-tag="div" lynx-unique-id="32" id="target-70"><div lynx-tag="div" lynx-unique-id="33" id="target-69"><div lynx-tag="div" lynx-unique-id="34" id="target-68"><div lynx-tag="div" lynx-unique-id="35" id="target-67"><div lynx-tag="div" lynx-unique-id="36" id="target-66"><div lynx-tag="div" lynx-unique-id="37" id="target-65"><div lynx-tag="div" lynx-unique-id="38" id="target-64"><div lynx-tag="div" lynx-unique-id="39" id="target-63"><div lynx-tag="div" lynx-unique-id="40" id="target-62"><div lynx-tag="div" lynx-unique-id="41" id="target-61"><div lynx-tag="div" lynx-unique-id="42" id="target-60"><div lynx-tag="div" lynx-unique-id="43" id="target-59"><div lynx-tag="div" lynx-unique-id="44" id="target-58"><div lynx-tag="div" lynx-unique-id="45" id="target-57"><div lynx-tag="div" lynx-unique-id="46" id="target-56"><div lynx-tag="div" lynx-unique-id="47" id="target-55"><div lynx-tag="div" lynx-unique-id="48" id="target-54"><div lynx-tag="div" lynx-unique-id="49" id="target-53"><div lynx-tag="div" lynx-unique-id="50" id="target-52"><div lynx-tag="div" lynx-unique-id="51" id="target-51"><div lynx-tag="div" lynx-unique-id="52" id="target-50"><div lynx-tag="div" lynx-unique-id="53" id="target-49"><div lynx-tag="div" lynx-unique-id="54" id="target-48"><div lynx-tag="div" lynx-unique-id="55" id="target-47"><div lynx-tag="div" lynx-unique-id="56" id="target-46"><div lynx-tag="div" lynx-unique-id="57" id="target-45"><div lynx-tag="div" lynx-unique-id="58" id="target-44"><div lynx-tag="div" lynx-unique-id="59" id="target-43"><div lynx-tag="div" lynx-unique-id="60" id="target-42"><div lynx-tag="div" lynx-unique-id="61" id="target-41"><div lynx-tag="div" lynx-unique-id="62" id="target-40"><div lynx-tag="div" lynx-unique-id="63" id="target-39"><div lynx-tag="div" lynx-unique-id="64" id="target-38"><div lynx-tag="div" lynx-unique-id="65" id="target-37"><div lynx-tag="div" lynx-unique-id="66" id="target-36"><div lynx-tag="div" lynx-unique-id="67" id="target-35"><div lynx-tag="div" lynx-unique-id="68" id="target-34"><div lynx-tag="div" lynx-unique-id="69" id="target-33"><div lynx-tag="div" lynx-unique-id="70" id="target-32"><div lynx-tag="div" lynx-unique-id="71" id="target-31"><div lynx-tag="div" lynx-unique-id="72" id="target-30"><div lynx-tag="div" lynx-unique-id="73" id="target-29"><div lynx-tag="div" lynx-unique-id="74" id="target-28"><div lynx-tag="div" lynx-unique-id="75" id="target-27"><div lynx-tag="div" lynx-unique-id="76" id="target-26"><div lynx-tag="div" lynx-unique-id="77" id="target-25"><div lynx-tag="div" lynx-unique-id="78" id="target-24"><div lynx-tag="div" lynx-unique-id="79" id="target-23"><div lynx-tag="div" lynx-unique-id="80" id="target-22"><div lynx-tag="div" lynx-unique-id="81" id="target-21"><div lynx-tag="div" lynx-unique-id="82" id="target-20"><div lynx-tag="div" lynx-unique-id="83" id="target-19"><div lynx-tag="div" lynx-unique-id="84" id="target-18"><div lynx-tag="div" lynx-unique-id="85" id="target-17"><div lynx-tag="div" lynx-unique-id="86" id="target-16"><div lynx-tag="div" lynx-unique-id="87" id="target-15"><div lynx-tag="div" lynx-unique-id="88" id="target-14"><div lynx-tag="div" lynx-unique-id="89" id="target-13"><div lynx-tag="div" lynx-unique-id="90" id="target-12"><div lynx-tag="div" lynx-unique-id="91" id="target-11"><div lynx-tag="div" lynx-unique-id="92" id="target-10"><div lynx-tag="div" lynx-unique-id="93" id="target-9"><div lynx-tag="div" lynx-unique-id="94" id="target-8"><div lynx-tag="div" lynx-unique-id="95" id="target-7"><div lynx-tag="div" lynx-unique-id="96" id="target-6"><div lynx-tag="div" lynx-unique-id="97" id="target-5"><div lynx-tag="div" lynx-unique-id="98" id="target-4"><div lynx-tag="div" lynx-unique-id="99" id="target-3"><div lynx-tag="div" lynx-unique-id="100" id="target-2"><div lynx-tag="div" lynx-unique-id="101" id="target-1"><div lynx-tag="div" lynx-unique-id="102" id="target-0"><raw-text lynx-tag="raw-text" lynx-unique-id="103" text="0"></raw-text></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></template></lynx-view>"
`;
